hummingbot/strategy/pure_market_making/pure_market_making_config_map.py

```diff
@@ -331,6 +341,14 @@ def exchange_on_validated(value: str):
                   prompt="Enter pricing API URL >>> ",
                   required_if=lambda: pure_market_making_config_map.get("price_source").value == "custom_api",
                   type_str="str"),
+    "inventory_cost_allow_higher_bids":
```

Comment:
Why do we need this config variable? And why only for bids?
From what I understand the bid price will always be below inventory price (if inventory price is known), the bid price is inventory price minus spread.

Reply:
There are 2 reasons:
1. Over time with profit accumulation inventory price goes lower and lower, so bids can become too far from the mid price, see example here https://github.com/CoinAlpha/hummingbot/issues/2314#issuecomment-716409652
2. When market is in uptrend, you may still want to perform occasional buys.

Reply:
Personally, i don't think we should crater for exceptions like this. Reasons:
* Your reasons can be argued for `inventory_cost_allow_lower_asks` as well which we don't have, just the opposite direction.
* Should you consider to apply this only when the user is in profit? also, is it realised or unrealised profit?
*  If there is `inventory_cost_allow_lower_asks`, turning both it and `inventory_cost_allow_higher_bids` on wouldn't make any sense, since it simply neglects the feature entirely.

Reply:
Ok, I'll revert this option.